frontend/components/CommentForm/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { useForm } from 'react-hook-form'
+import { isObjectEmpty } from '@/utils'
+import CommentFormSchema from './schema'
+import { useEffect } from 'react'
+import FormAlert from '../molecules/FormAlert'
+import Button from '../atoms/Button'
+import { yupResolver } from '@hookform/resolvers/yup'
+
+export type FormValues = {
+    comment: string
+}
+
+type CommentFormProps = {
+    id: number | null
+}
+
+const CommentForm = ({ id }: CommentFormProps): JSX.Element => {
+    const {
+        register,
+        handleSubmit,
+        setValue,
+        formState: { errors },
+    } = useForm<FormValues>({
+        mode: 'onSubmit',
+        reValidateMode: 'onSubmit',
+        resolver: yupResolver(CommentFormSchema),
+    })
+
+    const onSubmit = (data: FormValues) => {
+        // id - you can use this to condition between update and create
+        if (isObjectEmpty(errors)) {
+            //Replace with graphql post request
+            console.log(data)
+        }
+    }
+
+    useEffect(() => {
+        register('comment', {})
+        setValue('comment', '')
+    }, [register])
```

Comment:
I don't think this is needed anymore as you have already declared register in the text area.